Bonsai.Harp.CF/SyringePumpEvent.cs

```diff
@@ -46,7 +49,8 @@ string Description
                     case SyringePumpEventType.Direction: return "The state of the direction of motor's movement.";
                     case SyringePumpEventType.SwitchForward: return "The state of the forward switch.";
                     case SyringePumpEventType.SwitchReverse: return "The state of the reverse switch.";
-                    case SyringePumpEventType.Input: return "The state of the digital input 1.";
+                    case SyringePumpEventType.Input: return "The state of the digital input 0.";
+                    case SyringePumpEventType.Protocol: return "The running state of the protocol.";
```

Comment:
Is this always going to be just a boolean value indicating whether the protocol is running or not? If in the future there might be more states, it might be better to return a number or Enum type.